x-pack/plugins/security_solution/cypress/integration/detection_rules/custom_query_rule.spec.ts

```diff
@@ -218,18 +217,21 @@ describe('Custom query rules', () => {
             const initialNumberOfRules = rules.length;
             const expectedNumberOfRulesAfterDeletion = initialNumberOfRules - 1;
 
-            cy.get(SHOWING_RULES_TEXT).should('have.text', `Showing ${initialNumberOfRules} rules`);
+            cy.request({ url: '/api/detection_engine/rules/_find' }).then(({ body }) => {
```

Comment:
Nit: we could introduce a constant for this path.

The constant would be defined here:

https://github.com/elastic/kibana/blob/0212338e4e3ec9db5cfb4e93ad1850a156f0ea9b/x-pack/plugins/security_solution/common/constants.ts#L254-L263

We would use it in the tests and in the route handler:

https://github.com/elastic/kibana/blob/0212338e4e3ec9db5cfb4e93ad1850a156f0ea9b/x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/find_rules_route.ts#L26